packages/amplify_api/ios/Classes/ApiBridge.swift

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import Amplify
+import AmplifyPlugins
+import Flutter
+
+class ApiBridge {
+    
+    func get(request: RESTRequest, listener: ((AmplifyOperation<RESTOperationRequest, Data, APIError>.OperationResult) -> Void)?) -> RESTOperation?{
+        return Amplify.API.get(request: request, listener: listener)
+    }
+    
+    func put(request: RESTRequest, listener: ((AmplifyOperation<RESTOperationRequest, Data, APIError>.OperationResult) -> Void)?) -> RESTOperation?{
+        return Amplify.API.put(request: request, listener: listener)
+    }
+
+    func post(request: RESTRequest, listener: ((AmplifyOperation<RESTOperationRequest, Data, APIError>.OperationResult) -> Void)?) -> RESTOperation?{
+        return Amplify.API.post(request: request, listener: listener)
+    }
+    
+    func delete(request: RESTRequest, listener: ((AmplifyOperation<RESTOperationRequest, Data, APIError>.OperationResult) -> Void)?) -> RESTOperation?{
+        return Amplify.API.delete(request: request, listener: listener)
```

Comment:
Instead of `Amplify.API`, call through our abstraction, the category behavior, `ApiCategoryBehavior`. Same comment applies to both iOS & Android impls.

Reply:
Will do this as part of the test refactor PR as its related to that.